src/app/layout/AccountMenu/index.tsx

```diff
@@ -55,10 +55,10 @@ const AppVersion = ({ ...rest }) => {
         outline="none"
         color="gray.500"
         _hover={{ bg: 'gray.50' }}
-        _focus={{ bg: 'gray.50' }}
+        _focusVisible={{ bg: 'gray.500' }}
```

Comment:
I don't think `gray.500` is what you want here.

https://user-images.githubusercontent.com/3920615/180957117-d995e5cd-da16-4dfe-92e2-038e68144bec.mp4

Reply:
i fix this